public/index.pug

```diff
@@ -175,5 +175,7 @@ html(lang="en-US")
           title Twitter logo
           path(d=twitterIcon.path)
         span Share this on Twitter!
-
+      div(class="footer-repo")
+          p. 
```

Comment:
Minor but could you remove the trailing space here before someone else's editor (e.g. mine) removes it? (I really thought Prettier would take care of this :unamused:)

```suggestion
          p.
```

Reply:
yes, I don't know what happened there.